crates/sui-framework/sources/governance/voting_power.move

```diff
@@ -142,6 +143,7 @@ module sui::voting_power {
         let total = 0;
         while (i < len) {
             let voting_power = validator::voting_power(vector::borrow(v, i));
+            assert!(voting_power > 0, EInvalidVotingPower);
```

Comment:
Hmm I realized we only start doing low stake departures after `governance_start_epoch` so _technically_ this could be true before `governance_start_epoch`. @bmwill with the newest / planned changes in genesis, will all validators have at least 25M from the start?

Reply:
If the answer is yes, we should always do low stake departure even before governance_start_epoch

Reply:
@emmazzz yes all validators will have sufficient stake at genesis once the new genesis flow is finalized